src/ctia/bundle/routes.clj

```diff
@@ -67,9 +67,28 @@
     :read-casebook
     :list-casebooks})
 
+(defn- keyword->str [key] (str (if (namespace key) "/" "") (name key)))
+
+(defn- entity->bundle-keys
+  "For given entity key returns corresponding keys that may be present in Bundle schema.
+  e.g. :asset => [:assets :asset_refs]"
+  [entity-key]
+  (let [{:keys [entity plural]} (get (entities/all-entities) entity-key)
+        kw->snake-case-str      (fn [kw] (-> kw keyword->str (str/replace #"-" "_")))]
+    [(-> plural kw->snake-case-str keyword)
+     (-> entity kw->snake-case-str (str "_refs") keyword)]))
+
+(s/defn prep-bundle-schema :- s/Any
```

Comment:
In addition of your route level test could you add a unit test for this function?
Testing at the most granular level has the advantage of:
- easing the debugging
- better cover edge cases (which are harder to combine at a higher level)

Reply:
Take a look: 57fe0cafefb93d1d4b66a462698939e5fa9250d1